src/Prefixer.js

```diff
@@ -110,13 +110,13 @@ export function prefix (value, length, children) {
 				break
 		// display: (flex|inline-flex|grid|inline-grid)
 		case 6444:
-			switch (charat(value, strlen(value) - 3 - (~indexof(value, '!important') && 10))) {
+			switch (charat(value, strlen(value) - 3 - (~indexof(value, ' !important') ? 11 : ~indexof(value, '!important') && 10))) {
```

Comment:
this probably won't work for `value          !important`

Reply:
Fixed. And related test cases are added.